### ISSUE TITLE
[Feature request] Option to not dim unloaded tabs

### ISSUE THREAD

**Issue by Zeioth** [mbnuqw/sidebery#1377]
## Description
It would be great to have a option to disable the dim effect on unloaded tabs. I'm not interested in this information and it is distracting.

![screenshot_2023-12-03_09-40-27_245516621](https://github.com/mbnuqw/sidebery/assets/3357792/52f4d267-dafd-451c-8547-1aaf9e8318af)

**Comment by llc0930** [comment]
```
.Tab[data-discarded=true] .body {
	opacity: 1;
}

.Tab[data-discarded=true]>.body>.fav {
	opacity: 1;
}

.Tab[data-discarded=true]>.body>.t-box {
	opacity: 1;
}
```

**Comment by Montagnard11** [comment]
I knew I wasn't the only one...

**Comment by mbnuqw** [comment]
Resolved in #1553. From v5.2.0.4 you can change `--tabs-discarded-title-opacity` and `--tabs-discarded-favicon-opacity` in Sidebery settings > Styles editor > Tabs.
